main.js

```diff
@@ -236,6 +260,6 @@ const init = () => {
   } else if (window.location.pathname === '/projects.html') {
     projectBuilder(users);
   } 
-}
```

Comment:
Change init conditional to have else = pinnedRepoBuilder.

const init = () => {
  handleButtonEvents();
  if (window.location.pathname === "/packages.html") {
    createPackage(users); 
  } else if (window.location.pathname === "/index.html") {
    pinnedRepoBuilder(users);
  } else if (window.location.pathname === '/projects.html') {
    projectBuilder(users);
  } else {
    pinnedRepoBuilder(users);
  }
};

init();